src/routes/postSkipSegments.js

```diff
@@ -96,15 +95,51 @@ async function autoModerateSubmission(submission, callback) {
                 if (duration == 0) {
                     // Allow submission if the duration is 0 (bug in youtube api)
                     return false;
-                } else if ((submission.endTime - submission.startTime) > (duration/100)*80) {
-                    // Reject submission if over 80% of the video
-                    return "Sponsor segment is over 80% of the video.";
-                } else {
+                }
+
+                for (const segment of segments) {
```

Comment:
Are submissions one at a time now?

Reply:
It was. I made another comment where it was being called from. It should be called from the first for loop

Reply:
I fixed these merge issues